features/hq/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,28 @@
+World Helpers
+
+Given(/^I am a new, authenticated user$/) do
+  create_admin_user
+  login
+end
+
+Given(/^I am an unauthenticated user$/) do
+  logout
+end
+
+When(/^I (?:go to|am on) the "([^"]*)" page for the "([^"]*)" role$/) do  |page, role|
```

Comment:
That is interesting and it feels right.
1. We should only use BDD to test scenarios that bring value to the business of OSRA. (ex. authentication, create partners, sponsors, orpahns, start/terminate sponsorships, import orphan lists). This is somehow telling to not focus on "if every feature works" (happy/sad paths ...) and only focus on what the client needs this application for.

2. Instead of doing step by step features for every scenario we should use more general features and in step definitions use the helper methods (from step_helpers.rb) to keep the code DRY.

Example:
Instead of
```
Given I am a new, authenticated user
When I am on the "Partners" page
And I click "Add Partner"
And I fill in name with "John"
And I fill in city with "GB"
...
And I click "Create"
Then I should be on the "Partner" page for "John"
And I should see "Successfully created"
And I should see "John"
```

This scenario should look like:
```
Given I am a new, authenticated user
Then I should be able to create partners
```

or `I should be able to create partners from "create partners" page` if there are multiple places where partners can be created.

Then the step definition for `Then I should be able to create partners` should look like this:
```
Then(/^I should be able to create partners$/) do
  visit_partners_page
  click "Add Partner"
  expect(current_path).to be(new_partner_path)
  fill_in_partner_form_fields_with(:partner)
  click "Create partner"
  expect(current_path).to be(partner_path(1))
  expect(response).to have_content("Successfully created")
  expect(response).to have_content(partner.name)
end
```

PS: even if our client is not engaged in our cucumber tests and by that I mean we don't really need them, I think we should still build these tests (at a minimum/high level) just for practice/experience. And it also helps future new developers learn how OSRA works.